backend/src/openstates-api.js

```diff
@@ -0,0 +1,23 @@
+const URL_BASE = "https://v3.openstates.org";
+
+/** Construct URL to OpenStates legislator/legislation API
+ *
+ * https://open.pluralpolicy.com/
+ * Docs: https://docs.openstates.org/api-v3/
+ *
+ * @param {string} path
+ * @param {object} params
+ */
+export const openStatesApi = (path, params = {}) => {
+
+  // Add API key to all requests
+  params.apikey = process.env.OPEN_STATES_KEY;
+
+  // Note: currently we only support NY
+  // Note: this is case-sensitive (some fields, eg name, are not case-sensitive)
+  params.jurisdiction = "New York";
+  params.page = params.page || 1;
+  params.per_page = params.per_page || 10;
```

Comment:
If this is the call to get all reps in New York, why the the per-page limit set to 10? Don't we need all of them? Maybe I'm missing context from another file 😅

Reply:
It's the current default value: 
https://v3.openstates.org/docs#/people/people_search_people_get

I should probably add a comment.